CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
 - Do not edit this file directly. This file is auto-generated using [changie](https://github.com/miniscruff/changie). For details on how to document a change, see [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#adding-changelog-entry)
 
 
+## dbt-core 1.0.5-rc2 - April 08, 2022
+### Fixes
+- Catch more cases to retry package retrieval for deps pointing to the hub.  Also start to cache the package requests. ([#4849](https://github.com/dbt-labs/dbt-core/issues/4849), [#4982](https://github.com/dbt-labs/dbt-core/pull/4982))
+Contributors:
```

Comment:
@emmyoop is this correct where we always list the Contributor section even w/o Contributors? That's ok if that's the case, I just want to make sure it's a deliberate choice